packages/smart-accounts/src/configs.ts

```diff
@@ -0,0 +1,60 @@
+import * as common from '@protocolink/common';
+
+export enum SmartAccountId {
+  PORTUS = 'portus',
+}
+
+export interface SmartAccount {
+  id: string;
+  executor: string;
+}
+
+export interface Config {
+  chainId: number;
+  smartAccounts: SmartAccount[];
+}
+
+export const configs: Config[] = [
+  {
+    chainId: common.ChainId.optimism,
+    smartAccounts: [
+      {
+        id: SmartAccountId.PORTUS,
+        executor: '0xdD408b8eFb837EdeF8e6192Ed19f0dbEB7B79383',
+      },
+    ],
+  },
+];
+
+export const [supportedChainIds, configMap, executorMap, executorIdMap] = configs.reduce(
```

Comment:
```suggestion
export const [supportedChainIds, configMap, smartAccountMap, smartAccountIdMap] = configs.reduce(
```

Please also update `export function getXXX` naming